content/project/utilities/python-apple-support/contents.lr

```diff
@@ -6,7 +6,7 @@ maturity: beta
 ---
 platforms: macOS, iOS, tvOS, watchOS
 ---
-project_type: support
+project_type: utility
 ---
 short_description: A meta-package for building a version of Python that can be embedded into a macOS, iOS, tvOS or watchOS project.
 ---
```

Comment:
Mental note that if we're going to highlight this package (and I think we should), I should probably provide some copy so that this isn't an incomplete stub.